back/boxtribute_server/graph_ql/resolvers.py

```diff
@@ -332,18 +331,32 @@ def resolve_create_qr_code(_, info, box_label_identifier=None):
 
 @mutation.field("createBox")
 @convert_kwargs_to_snake_case
-def resolve_create_box(_, info, box_creation_input):
+def resolve_create_box(*_, box_creation_input):
     authorize(permission="stock:write")
-    box_creation_input["created_by"] = g.user["id"]
-    return create_box(box_creation_input)
+    return create_box(
+        product_id=box_creation_input["product_id"],
+        location_id=box_creation_input["location_id"],
+        comment=box_creation_input["comment"],
+        items=box_creation_input["items"],
+        size_id=box_creation_input["size_id"],
+        qr_code_code=box_creation_input.get("qr_code"),
```

Comment:
I wouldn't mind to have the detailed parameters hidden away by extracting `**box_creation_input` on the resolver level here. The explicit passing of parameters has a bit of redundancy and verbosity. If one wants to know the exact signature of `create_box` they can inspect its definition. I've used this approach in the resolvers for all `create`/`update` operations.

Reply:
I am fine with this, but it basically limits us for now to use the same parameter names everywhere (which might not be too bad). E.g. see `qr_code` vs `qr_code_code`.

Reply:
Mhh ya I saw that you did a remapping. Maybe a comment in the signature of `create_box` helps? e.g. `qr_code=None  # code string of a QrCode object` or similar

Reply:
I just renamed it back, it is pretty clear that this is not a peewee object from the context here.